output.tf

```diff
@@ -2,3 +2,9 @@ output "service_account_name" {
   value       = local.service_account_name
   description = "The Service Account name created for the integration"
 }
+
+output "service_account_private_key" {
```

Comment:
I had removed this because it causes a warning on the deployment - this is a sensitive value and TF no longer likes outputting it.  Do we want to keep it in anyway?

Reply:
Let us keep it for now. Great question though! 👍🏽